hstream-sql/etc/SQL-v1.cf

```diff
@@ -336,4 +337,4 @@ ArrayFuncMax.       ScalarFunc ::= "ARRAY_MAX"       "(" ValueExpr
 ArrayFuncMin.       ScalarFunc ::= "ARRAY_MIN"       "(" ValueExpr               ")" ;
 ArrayFuncSort.      ScalarFunc ::= "ARRAY_SORT"      "(" ValueExpr               ")" ;
 
-coercions ValueExpr 8 ;
+coercions ValueExpr 10 ;
```

Comment:
Is this `9` or `10`? I do not find `ValueExpr10`... https://bnfc.readthedocs.io/en/latest/lbnf.html#coercions

Reply:
10 or 9 is both okay, with between keywords introduced it will turns to 10. let us use 9 for now?